test/blackbox-tests/test-cases/foreign-stubs/fs-mode-dependent/dune

```diff
@@ -0,0 +1,3 @@
+(cram
+ (applies_to :whole_subtree)
+ (alias fsmd))
```

Comment:
You already have `./dune.exe runtest test/blackbox-tests/test-cases/foreign-stubs`. Isn't it enough?